lumibot/brokers/tradier.py

```diff
@@ -24,15 +26,18 @@ class Tradier(Broker):
     POLL_EVENT = PollingStream.POLL_EVENT
 
     def __init__(
-        self,
-        config=None,
-        account_number=None,
-        access_token=None,
-        paper=None,
-        max_workers=20,
-        connect_stream=True,
-        data_source=None,
-        polling_interval=5.0,
+            self,
+            config=None,
+            account_number=None,
+            access_token=None,
+            paper=None,
+            connect_stream=True,
+            data_source=None,
+            polling_interval=5.0,
+
+            # Need sequential order submission for Tradier becuase it is very strict that buy orders exist
```

Comment:
can we do this with all brokers? i think all of them should have sequential orders, no?